apis/opts/diskquota_test.go

```diff
@@ -19,7 +19,7 @@ func TestParseDiskQuota(t *testing.T) {
 		{name: "test2", args: args{diskquota: []string{"foo=foo=foo"}}, want: nil, wantErr: true},
 		{name: "test3", args: args{diskquota: []string{"foo"}}, want: map[string]string{"/": "foo"}, wantErr: false},
 		{name: "test4", args: args{diskquota: []string{"foo=foo"}}, want: map[string]string{"foo": "foo"}, wantErr: false},
-		{name: "test5", args: args{diskquota: []string{"foo = foo"}}, want: map[string]string{"foo": "foo"}, wantErr: true},
+		{name: "test5", args: args{diskquota: []string{"foo= foo"}}, want: map[string]string{"foo": "foo"}, wantErr: false},
```

Comment:
it had better to unify key=value format, for instance, to remove useless whitespace.

Reply:
Yes, i thinks so.
So, do i delete this test case?

Reply:
This should also be a test case, I think `ParseDiskQuota` should test it valid or invalid whether has space in string slice @hellolijj @chuanchang